### PR TITLE
ds-prep-python-functions- Tiffany Lin

### PR DESCRIPTION
**Pull request by tiffanylin706**
截圖 2022-04-25 下午3 08 09

### CODE REVIEW

File: ds-prep-python-functions/main.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+print  (cube(5))
+
+def absolute_difference (a, b) :
+  return abs ( a,b )
+print ("absolute difference:", (abs (14-11)))
+print ("absolue difference:", (abs (13-40)))
+
+def squared_difference (a, b):
+  return ((abs (a,b))**2)
+print ("square_difference:", ((abs(14-11))**2))
```
Comment (TimelikeClosure) [inline ds-prep-python-functions/main.py]: Please make sure your `squared_difference` function is properly calculating the square of the difference, so you're not forced to do the math outside the function.

File: ds-prep-python-functions/main.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+def hours_to_minutes (hours):
+  return (hours*60)
+print ("hours_to_minutes:", hours_to_minutes (3.5))
+print ("hours_to_minutes:", hours_to_minutes (12))
+
+def get_circumference (radius):
+  return (radius* 6.283185307179586)
+print ("get_circumference:", get_circumference (9.2))
```
Comment (TimelikeClosure) [inline ds-prep-python-functions/main.py]: Please call these with _keyword_ arguments only:

---

image

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: ds-prep-python-functions/main.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+def cube (base):
+  return (base**3)
+print  (cube(3))
+print  (cube(5))
+
+def absolute_difference (a, b) :
+  return abs ( a,b )
+print ("absolute difference:", (abs (14-11)))
+print ("absolue difference:", (abs (13-40)))
```
Comment (TimelikeClosure) [inline ds-prep-python-functions/main.py]: Please call the `absolute_difference` function to get these values. Make sure your `absolute_difference` function definition is using `abs` correctly: https://www.w3schools.com/python/ref_func_abs.asp

File: ds-prep-python-functions/main.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+def squared_difference (a, b):
+  return ((abs (a,b))**2)
+print ("square_difference:", ((abs(14,11))**2))
```
Comment (TimelikeClosure) [inline ds-prep-python-functions/main.py]: Like above, please make sure you're calling the `squared_difference` function to get the value, not calculating it manually outside the function definition.

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Review by TimelikeClosure — APPROVED** [review]

**Comment by tiffanylin706** [comment]
截圖 2022-04-25 下午7 54 07

**Pull request merged**